dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2291,6 +2291,38 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_SCHEDULER_RESTORE_JOB_FROM_JOURNAL =
+      booleanBuilder(Name.MASTER_SCHEDULER_RESTORE_JOB_FROM_JOURNAL)
```

Comment:
I'm OK with adding an option but I would prefer to have fewer properties and one of the big complaints from customers is that we have too many properties. In which situation we would benefit if this is false? The only situation I can think of is in test env.

Reply:
i guess we discussed this offline..... I proposed to stop restoring incomplete jobs from the journals which you showed your concorn with.... That's why i added an option for this. Mind reminding me if there's a better way?

When i used this distributed tool in a real cluster, i've noticed that after restarting the cluster, the job just starts unexpectedly and created some operaation challenges.